### PR TITLE
Update cheatcodes docs

### PR DESCRIPTION
**Pull request by Draggu**
## Introduced changes

- Updates docs for `cheat_execution_info` and it's helpers
- Remove old cheatcodes docs
- Updates changelog

Follow-up for #2102

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Forge
+
+#### Removed
+
+`prank`, `warp`, `roll`, `elect`, `spoof` cheatcodes
```
Comment (Arcticae) [inline CHANGELOG.md:14]:
```suggestion
- `prank`, `warp`, `roll`, `elect`, `spoof` cheatcodes
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+#### Added
+
+`cheat_execution_info` cheatcode and per variable helpers for it
```
Comment (Arcticae) [inline CHANGELOG.md:18]:
```suggestion
- `cheat_execution_info` cheatcode and per variable helpers for it
```

File: docs/src/appendix/cheatcodes.md
Comment (Arcticae) [inline docs/src/appendix/cheatcodes.md:1]: That page is unreadable as is now, we should organize it somehow (into sections perhaps)

File: docs/src/appendix/cheatcodes/account_deployment_data.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+> `fn cheat_account_deployment_data_global(account_deployment_data: Span<felt252>)`
+
+Changes the transaction account deployment data for all targets.
+
+## `start_cheat_account_deployment_data`
+> `fn start_cheat_account_deployment_data(target: ContractAddress, account_deployment_data: Span<felt252>)`
+
+Changes the transaction account deployment data for the given target.
+
+# `stop_cheat_account_deployment_data`
```
Comment (Arcticae) [inline docs/src/appendix/cheatcodes/account_deployment_data.md:20]: All of the pages have inconsistent title sizes

File: docs/src/appendix/cheatcodes/version.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# `version`
+
+Cheatcodes modifying `version`:
```
Comment (Arcticae) [inline docs/src/appendix/cheatcodes/version.md:3]: this is ambiguous to the reader - which `version` does it refer to?

**Review by Arcticae — COMMENTED** [review]
We should organize the appendix in a smart way, so that the reader knows that particular cheatcodes refer to parts of `TxInfo` structures.

Preferably, i would also add a guide explaining which cheatcode group to use, so that easier navigation through the docs is possible

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Forge
+
+#### Removed
+
+- `prank`, `warp`, `roll`, `elect`, `spoof` cheatcodes
```
Comment (Arcticae) [inline CHANGELOG.md:14]: It seems that we removed it and not provided the alternative. We should also provide some rationale here so we can refer to it

Comment (Draggu) [reply CHANGELOG.md]: Fixed

**Review by piotmag769 — APPROVED** [review]

File: docs/src/appendix/cheatcodes/version.md
Comment (Arcticae) [inline docs/src/appendix/cheatcodes/version.md:1]: this file name needs changing

File: docs/src/appendix/cheatcodes/version.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# `version`
```
Comment (Arcticae) [inline docs/src/appendix/cheatcodes/version.md:1]: the title needs changing

**Review by Arcticae — APPROVED** [review]

**Pull request merged**